frontend/awx/access/credential-types/hooks/useCredentialTypeActions.tsx

```diff
@@ -11,21 +11,19 @@ import {
 import { useOptions } from '../../../../common/crud/useOptions';
 import { awxAPI } from '../../../common/api/awx-utils';
 import { IAwxView } from '../../../common/useAwxView';
-import { CredentialType } from '../../../interfaces/CredentialType';
 import { ActionsResponse, OptionsResponse } from '../../../interfaces/OptionsResponse';
 import { AwxRoute } from '../../../main/AwxRoutes';
 import { useDeleteCredentialTypes } from './useDeleteCredentialTypes';
+import { CredentialType } from '../../../interfaces/CredentialType';
 
 export function useCredentialTypeToolbarActions(view: IAwxView<CredentialType>) {
```

Comment:
We can leverage passing the same on delete credential prop here as well.
```suggestion
export function useCredentialTypeToolbarActions(onCredentialTypesDeleted: (credentialType: CredentialType[]) => void) {
```